File: ArrayQuestion/src/SecondLargest.java
public class SecondLargest {

    public static void main(String[] args) {

        int[] arr = {5, 4, 6, 1, 3, 2};

        secondLargest(arr);


    }

    static void secondLargest(int[] arr) {

        int Max = arr[0];
        int second_max = Integer.MIN_VALUE;

        for (int i = 0; i < arr.length; i++) {
            if (arr[i] > Max) {
                Max = arr[i];
            }
        }

        for (int j = 0; j < arr.length; j++) {
            if ( arr[j] != Max && arr[j] > second_max) {
                second_max = arr[j];
            }
        }

        System.out.println(second_max);
    }
}

File: ArrayQuestion/secondlargest.cpp
#include <iostream>
#include <vector>
#include <math.h>

using namespace std;


void secondLargest(vector<int> arr){

  int max = INT_MIN;
  int second_max = INT_MIN;

  for(int i  = 0; i < arr.size(); i++){
    if(arr[i] > max){
      max = arr[i];
    }
  }

  for(int i  = 0; i < arr.size(); i++){
    if(arr[i] != max && arr[i] > second_max){
      second_max = arr[i];
    }
  }

  cout<<second_max;

  
}


int main(){

  vector<int> arr = {2,3,5,8,5,9};

  secondLargest(arr);
}